.github/workflows/mobile_release.yml

```diff
@@ -0,0 +1,105 @@
+name: mobile_release
+
+on:
+  workflow_dispatch:
+  schedule:
+  # Mondays at 1pm UTC (8am EST)
+  - cron: "0 13 * * 1"
+
+jobs:
+  android_release_artifacts:
+    if: github.repository == 'envoyproxy/envoy'
+    name: android_release_artifacts
+    runs-on: ubuntu-20.04
+    timeout-minutes: 120
+    container:
+      image: envoyproxy/envoy-build-ubuntu:mobile-458cb49ca2013c0ccf057d00ad1d4407920c4e52
+      env:
+        CC: /opt/llvm/bin/clang
+        CXX: /opt/llvm/bin/clang++
+    steps:
+    - uses: actions/checkout@v3
+      with:
+        fetch-depth: 0
+    - name: Add safe directory
+      run: git config --global --add safe.directory /__w/envoy/envoy
+    - name: 'Build envoy.aar distributable'
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      working-directory: mobile
+      run: |
+        version="0.5.0.$(date '+%Y%m%d')"
+        ./bazelw build \
+          $([ -z $GITHUB_TOKEN ] || echo "--config=remote-ci-linux-clang") \
+          --remote_header="Authorization=Bearer $GITHUB_TOKEN" \
+          --fat_apk_cpu=x86,x86_64,armeabi-v7a,arm64-v8a \
+          --define=pom_version="$version" \
+          --config=release-android \
+          --linkopt=-fuse-ld=lld \
+          //:android_dist
+    - name: 'Tar artifacts'
+      run: tar -czvf envoy_android_aar_sources.tar.gz bazel-bin/library/kotlin/io/envoyproxy/envoymobile/envoy.aar bazel-bin/library/kotlin/io/envoyproxy/envoymobile/envoy-pom.xml bazel-bin/library/kotlin/io/envoyproxy/envoymobile/envoy-sources.jar bazel-bin/library/kotlin/io/envoyproxy/envoymobile/envoy-javadoc.jar
+      working-directory: mobile
+    - uses: actions/upload-artifact@v2
+      with:
+        name: envoy_android_aar_sources
+        path: mobile/envoy_android_aar_sources.tar.gz
+  android_release_deploy:
+    name: android_release_deploy
+    needs: android_release_artifacts
+    runs-on: ubuntu-22.04
```

Comment:
we use `ubuntu-20.04` in some of the other places in this file. Can we unify and use one version?

Reply:
I'd rather not risk breaking this, since I tested with 22.04 and won't have time to test another deployment with 20.04 then revert the temporary changes to deploy from this PR.

If you'd like you could take that as a follow-up, or I can look into it when I'm back from my leave.